solidity/dashboard/src/components/Footer.jsx

```diff
@@ -14,9 +14,9 @@ const Footer = () => {
     <footer className="footer">
       <ul className="footer__links">{links.map(renderFooterLinkItem)}</ul>
       <div className="footer__signature">
-        A Thesis* Build <br />
-        © 2020 Keep, SEZC <br />
-        All Rights Reserved.
+        <p>A Thesis* Build</p>
+        <p>&#169; 2020 Keep, SEZC</p>
```

Comment:
Just to be clear---I'm very comfortable with us using Unicode here rather than HTML encodings. Not sure if eslint yells about it though. I find using Unicode directly is much clearer when editing later, and we've generally reached a point where everyone understands UTF-8 and all our files are saved as UTF-8.

Reply:
<img width="173" alt="Screen Shot 2020-10-06 at 09 48 45" src="https://user-images.githubusercontent.com/8245/95210106-22419680-07b9-11eb-96b3-51589206ba70.png">

Just flagging: looks like `line-height` is making these still be considerably more spaced out vertically than the design:

<img width="178" alt="Screen Shot 2020-10-06 at 09 49 12" src="https://user-images.githubusercontent.com/8245/95210154-31284900-07b9-11eb-92f7-eea089c74df6.png">

Reply:
> Just flagging: looks like line-height is making these still be considerably more spaced out vertically than the design

Right thanks for that! Probably it will be resolved with https://github.com/keep-network/keep-core/issues/2065, as I can see in Figma, typography styles have changed. I'll update all in a followup PR.